keps/693-multikueue/README.md

```diff
@@ -189,23 +197,40 @@ type MultiKueueClusterStatus {
 }
 ```
 
-MultiKueue controller will monitor all cluster definitions and maintain 
-the Kube clients for all of them. Any connectivity problems will be reported both in
-MultiKueueCluster status as well as AdmissionCheckStatus and Events. MultiKueue controller 
-will make sure that whenever the kubeconfig is refreshed, the appropriate 
-clients will also be recreated.
-
-Creation of kubeconfig files is outside of the MultiKueue scope, and is cloud
-provider/environment dependant.
-
 MultiKueue controller, when pushing workloads to the worker clusters, will use the same 
 namespace and local queue names as were used in the management cluster. It is user's 
 responsibility to set up the appropriate namespaces and local queues.
 Worker ClusterQueue definitions may be different than in the management cluster. For example,
 quota settings may be specific to the given location. And/or cluster queue may have different
 admission checks, use ProvisioningRequest, etc.
 
-When distributing the workloads across clusters MultiKueue controller will first create
+### Subcomponents
```

Comment:
In the API section could you please include the global API configuration for the MultiKueue AC?

Reply:
As in yaml objects to be to be created in the cluster? , Like we have here https://kueue.sigs.k8s.io/docs/tasks/manage/setup_multikueue/#create-a-sample-setup ?

Reply:
I meant global config API as here: https://github.com/kubernetes-sigs/kueue/blob/3f46a781dd43b74ec64514f6cee7cb7c32ffbd3b/apis/config/v1beta1/configuration_types.go#L211C6-L231

Reply:
Having the global API for MultiKueue discussed in KEP will help us to make informed decisions answering questions like: where to place some user-facing APIs? Is it global or per cluster, per admission check? 

I think we will have such considerations when designing the extension point for the custom jobs in MultiKueue, when thinking if this should it be global, or per admission check.

We added the global configuration to KEPs it in the past, for example for [wait for pods ready](https://github.com/kubernetes-sigs/kueue/tree/main/keps/349-all-or-nothing#kueue-configuration-api).

Reply:
In other words - both global API and per cluster queue are admin facing APIs for MultiKueue. I think the distinction if they are global, or per some scope is secondary.

Reply:
added